From 6a522b757ab283d709c8e3c02d53ea0c6819f181 Mon Sep 17 00:00:00 2001
From: Zhiyuan Lv <zhiyuan.lv@intel.com>
Date: Thu, 21 May 2015 14:38:07 +0800
Subject: [PATCH 083/403] Remove the forcewake workaround

It is not needed after we adding 0xfdc handling in force wake operations.

Signed-off-by: Zhiyuan Lv <zhiyuan.lv@intel.com>
---
 drivers/gpu/drm/i915/vgt/vgt.c |    9 ---------
 1 file changed, 9 deletions(-)

diff --git a/drivers/gpu/drm/i915/vgt/vgt.c b/drivers/gpu/drm/i915/vgt/vgt.c
index 6de68bd..0f6a589 100644
--- a/drivers/gpu/drm/i915/vgt/vgt.c
+++ b/drivers/gpu/drm/i915/vgt/vgt.c
@@ -796,15 +796,6 @@ static int vgt_initialize(struct pci_dev *dev)
 
 	printk("vgt_initialize succeeds.\n");
 
-	/* FIXME
-	 * always enable forcewake. It was found that forcewake
-	 * operation is one of the stability issue for running
-	 * windows guest. Before having a decent fix, we will
-	 * always enable force wake for Broadwell.
-	 */
-	if (IS_BDW(pdev))
-		vgt_force_wake_get();
-
 	return 0;
 err:
 	printk("vgt_initialize failed.\n");
-- 
1.7.10.4

